spec/dummy-app/spec/features/sign_in_spec.rb

```diff
@@ -7,16 +7,19 @@
 
   let!(:user_agent) { 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/78.0.3904.108 Safari/537.36' }
   let!(:accept_header) { 'text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8' }
+  let!(:accept_language) { 'en' }
```

Comment:
I think is a consequence of calling `.first` on the header. Can we use an example for the whole string?
`en-GB,en-US;q=0.9,en;q=0.8`